### PR TITLE
Rename to use underscores

### LINKED ISSUE #2: Renaming?
It seems we used `-` instead of `_`  ? It seems like their [`pip` install]( https://pypi.python.org/pypi/sphinx-bootstrap-theme#installation ) instructions recommend `sphinx_bootstrap_theme`. Should we be renaming the package @conda-forge/sphinx-bootstrap-theme ?

cc @conda-forge/core

### PR DESCRIPTION
**Pull request by ericdill**
The sphinx bootstrap theme package in pypi uses underscores and their
docs recommend using underscores. It was my mistake to use hyphens here
on conda-forge

Closes #2

@jakirkham I'm assuming we should remove the hyphenated packages on
anaconda.org/conda-forge?

### CODE REVIEW

**Comment by jakirkham** [comment]
> @jakirkham I'm assuming we should remove the hyphenated packages on
anaconda.org/conda-forge?

Not sure if that is a good idea. People may be depending on them.

**Comment by jakirkham** [comment]
FYI have enabled CircleCI for PRs on this repo again. So you are welcome to disable CircleCI on your fork, @ericdill .

**Comment by jakirkham** [comment]
> The sphinx bootstrap theme package in pypi uses underscores and their
docs recommend using underscores. It was my mistake to use hyphens here
on conda-forge

An unfortunate and confusing matter always, the way you are fixing it here is correct. Thanks for updating it, @ericdill .

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,9 +26,18 @@ test:
     - sphinx_bootstrap_theme
 
 about:
-  home: http://ryan-roemer.github.com/sphinx-bootstrap-theme/README.html
-  license: MIT License
+  home: http://ryan-roemer.github.com/sphinx-bootstrap-theme/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipe/meta.yaml:32]: Normally this would require a build number bump, but the rename has the same effect.

**Comment by jakirkham** [comment]
LGTM. Is this ready to go?

**Comment by ericdill** [comment]
@jakirkham  Thanks for circling back to this.

I don't remember why this was added

```
 build:
   preserve_egg_dir: True
```

are you able to discern if this is still needed?

Otherwise this should be good to go

**Comment by ericdill** [comment]
We can circle back to this preserve_egg_dir thing in another PR if it is no longer needed and/or harmful. Thanks for the bump on this @jakirkham

**Pull request merged**